### ISSUE TITLE
Another Captcha format

### ISSUE THREAD

**Issue by Gi0vaniJB** [SecreSwalowtail/Nostalgia2CaptchaResolver#2]
Do you have any source code capable of solving this Captcha model?
![Screenshot_1](https://github.com/SecreSwalowtail/Nostalgia2CaptchaResolver/assets/141739026/54aed68e-e294-4922-9b7a-870bb257d0ca)

**Comment by SecreSwalowtail** [comment]
> Do you have any source code capable of solving this Captcha model? ![Screenshot_1](https://private-user-images.githubusercontent.com/141739026/303915815-54aed68e-e294-4922-9b7a-870bb257d0ca.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc2NjUyMDUsIm5iZiI6MTcwNzY2NDkwNSwicGF0aCI6Ii8xNDE3MzkwMjYvMzAzOTE1ODE1LTU0YWVkNjhlLWUyOTQtNDkyMi05YjdhLTg3MGJiMjU3ZDBjYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIxMVQxNTIxNDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mODA4NjQyODRmZmY5ODM2YWRkZTE2YjkyYjY5MzY5ZTkzOGJhMjM5ZmM4ZTdiNTUzNDUyMGU3ZjBmMThjODc1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.MDQs0xA2NiGTTgokTG9Xy4zu8lc3pEDD6YIZBM55ZMA)

You need a solution like [2captcha](https://2captcha.com/) for captchas like the one you showed. The pricing I think is 1$ per 1k images. I can implement that for free if you are interested.

**Comment by Gi0vaniJB** [comment]
Yes, I am interested, this Captcha is increasingly present on current Metin2 servers, the solution will be very useful as well

**Comment by SecreSwalowtail** [comment]
> Yes, I am interested, this Captcha is increasingly present on current Metin2 servers, the solution will be very useful as well

secre11
Add me on discord and we can discuss further.

**Comment by Gi0vaniJB** [comment]
I added it

**Comment by PrinceOfFear** [comment]
voce conseguiu fazer esse tipo de captch ??

**Comment by failofwar** [comment]
Hello, following up on this. Was this ever added?
